Routes/Products.js

```diff
@@ -0,0 +1,19 @@
+const bodyParser = require('body-parser')
+const express = require('express')
+var router = express.Router()
+const { Product } = require('../models/Product.model')
+
```

Comment:
bodyparser is already been consumed in index.js file, here we don't need it again.
You can remove line 1 and line 7